CHANGELOG.md

```diff
@@ -10,10 +10,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
in your commit message title:

`🐛(courses) show an error banner when template is misused`